.env.example

```diff
@@ -2,7 +2,8 @@ TOKEN=get your discord token from https://discord.dev
 PREFIX=^
 RANDOMKEY=
 DEFAULTLANG=en_US
+LOG_LEVEL=info
 DB_NAME=lukas
-DB_HOST=localhost
+DB_HOST=docker.host.internal
```

Comment:
when using docker we should probably use the database from our stack so DB_HOST=db right?

Reply:
iirc you have to use docker.host.internal, but i'll try it